packages/esm-patient-registration-app/src/patient-registration/field/name/name-field.component.tsx

```diff
@@ -80,7 +80,6 @@ export const NameField = () => {
       id="middleName"
       name="middleName"
       labelText={t('middleNameLabelText', 'Middle Name')}
-      light
```

Comment:
Any specific reason for removing this one?

Reply:
It was causing warning in the test output and I don't think it is creating any effect. It is deprecated now.